### PR TITLE
components: add upload image component

### LINKED ISSUE #169: Adicionar componente de pré-visualização de imagens carrgadas
### Context
Adicionar componente de visualização de imagens carregados via componente de upload.

### Example
screenshot from 2018-07-12 16-41-01

### Description
```jsx
<UploadImage 
   name=""
   onChange={() => undefined}
   disabled={false}
/>
```

### Requirements
- [ ] Carregar imagens
- [ ] Gerar pré-visualizações de imagens

### PR DESCRIPTION
**Pull request by alexandrejuk**
resolve #169

### CODE REVIEW

File: src/UploadImage/index.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+import ThemeConsumer from '../ThemeConsumer'
+
+import Button from '../Button'
+import Input from '../Input'
+
+
+const ConsumeTheme = ThemeConsumer('UIUploadImage')
+
+class UploadImage extends Component {
+  constructor (props) {
```
Comment (uselessdev) [inline src/UploadImage/index.js]: this props is unnecessary here

File: src/UploadImage/index.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+import PropTypes from 'prop-types'
+import ThemeConsumer from '../ThemeConsumer'
+
+import Button from '../Button'
+import Input from '../Input'
+
+
+const ConsumeTheme = ThemeConsumer('UIUploadImage')
+
+class UploadImage extends Component {
```
Comment (uselessdev) [inline src/UploadImage/index.js]: I think this class component should be a pure function and just receive the props.

File: src/UploadImage/index.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import ThemeConsumer from '../ThemeConsumer'
+
+import Button from '../Button'
+import Input from '../Input'
+
```
[the remark this answers is not in the document]
Comment (alexandrejuk) [reply src/UploadImage/index.js]: I agree.

**Comment by augusto-jm-amaral** [comment]
@alexandrejuk I think the commit title is redundant, you can remove the "component" from the description.

**Comment by uselessdev** [comment]
Hi @alexandrejuk thank you.

Your PR is broken on CI because you need to update the storyshoot, you can do it by running this command `yarn test -- -u`.

I think the commit message can be better write, the context is where you make the changes or what you are adding, so I think a better commit message could be: `updalod-image: add component`.

I think the component should just transform image in a blob file, in this way we don't need to upload image before preview.

**Comment by alalbux** [comment]
This PR is out-of-date with master, could you @alexandrejuk rebase this with master?

**Pull request closed without merging**